### PR TITLE
CN synchronize topic

### PR DESCRIPTION
**Pull request by HuynhNguyen0811**

### CODE REVIEW

File: routes/admin.route.js [hunk 1]
```diff
@@ -15,13 +15,17 @@ export const config = {
...
 
-router.get('/detail/:id', async function (req, res) {
+router.get('/topicdetail/:id', async function (req, res) {
     const topicid = req.params.id
+    const topicname = await adminServices.getTopicName(topicid)
     const topic = await adminServices.getTopicDetail(topicid)
     const word = await adminServices.countWords(topicid)
     const test = await adminServices.getTest(topicid)
 
+    console.log(topicname[0].topicname)
```
Comment (quytrungg) [inline routes/admin.route.js:25]: @HuynhNguyen0811 console.log?

File: routes/admin.route.js [hunk 2]
```diff
@@ -15,13 +15,17 @@ export const config = {
...
+
     res.render('vwAdmin/topicdetail', {
+        topicname: topicname[0].topicname,
         topic: topic,
         num: word,
         test: test
```
Comment (quytrungg) [inline routes/admin.route.js:31]:
```suggestion
        topicname: topicname[0].topicname,
        topic,
        num: word,
        test,
```

**Review by quytrungg — APPROVED** [review]

File: routes/admin.route.js (hunk 1 above)
Comment (HuynhNguyen0811) [reply routes/admin.route.js:25]: I forgot to delete that :(

**Review by mugetsu0422 — APPROVED** [review]
noice

**Pull request merged**